internal/image/ova.go

```diff
@@ -117,6 +118,7 @@ func (o *Ova) generateIgnition() (string, error) {
 	}
 
 	butaneContent := strings.Replace(string(butaneTemplate), "@CONFIG_ID@", o.Id.String(), -1)
+	butaneContent = strings.Replace(butaneContent, "@SSH_KEY@", o.SshKey, -1)
```

Comment:
WDYT about using go templates instead of this method?

Reply:
Good idea. I've changed it. I've started to use syntax @NAME@ because at the begging I've used sed to do those things.